docs/deployment/setup_gardener.md

```diff
@@ -1,44 +1,412 @@
-# Deploying Gardener into a Kubernetes Cluster
+# How to Set Up a Gardener Landscape
 
-Similar to Kubernetes, Gardener consists out of control plane components (Gardener API server, Gardener controller manager, Gardener scheduler), and an agent component (gardenlet).
-The control plane is deployed in the so-called garden cluster, while the agent is installed into every seed cluster.
-Please note that it is possible to use the garden cluster as seed cluster by simply deploying the gardenlet into it.
+> [!IMPORTANT]
+> __DISCLAIMER__
+> This document outlines the building blocks used to set up a Gardener landscape. It is not meant as a comprehensive configuration guide but rather as a starting point. Setting up a landscape requires careful planning and consideration of dimensions like the number of providers, geographical distribution, and envisioned size.
+>
+>To make it more tangible, some choices are made to provide examples - specifically this guide uses OpenStack as infrastructure provider, Garden Linux to run any worker nodes, and Calico as CNI.
+> Providing working examples for all combinations of components is out of scope. For a detailed descriptions of components, please refer to the [documentation](https://gardener.cloud/docs/) and [API reference](https://gardener.cloud/docs/gardener/api-reference/).
 
-We are providing [Helm charts](../../charts/gardener) in order to manage the various resources of the components.
-Please always make sure that you use the Helm chart version that matches the Gardener version you want to deploy.
+## Target Picture
 
-## Deploying the Gardener Control Plane (API Server, Admission Controller, Controller Manager, Scheduler)
+A Gardener landscape consists of several building blocks. The basis is an existing Kubernetes cluster which is referred to as *runtime* cluster.
+Within this cluster another Kubernetes cluster is hosted, which is referred to as *virtual Garden* cluster. It is nodeless, meaning it does not host any pods as workload. The virtual Garden cluster is the management plane for the Gardener landscape and contains resources like `Shoots`, `Seeds` and so on.
 
-In order to deploy the control plane components, please first deploy [`gardener-operator`](../concepts/operator.md#deployment) and create a [`Garden` resource](../concepts/operator.md#garden-resources).
+Any new Gardener landscape starts with the deployment of the Gardener Operator and the creation of a `garden` namespace on the designated runtime cluster.
 
-> [!CAUTION]
-> Below approach is deprecated and will be removed after v1.135 of Gardener has been released (around beginning of 2026).
+With the operator in place, the `Garden` resource can be deployed. It is reconciled by the Gardener Operator, which will create the "virtual" Garden cluster with its `etcd`, `kube-api-server`, `gardener-apiserver`, and so on.
+The API server is exposed through a LoadBalancer `Service` and a DNS record pointing to its external IP address is created.
```

Comment:
Technically it is exposed via Istio, whose ingress is exposed via LoadBalancer service.

Reply:
addressed with a37e8e4b077f2b6ae111606124d82f0448377565